src/components/structures/MessagePanel.tsx

```diff
@@ -306,8 +306,14 @@ export default class MessagePanel extends React.Component<IProps, IState> {
         }
     }
 
+    private get hideSender(): boolean {
```

Comment:
I would prefer this to be a normal private method rather than a getter, but up to you.